eva_submission/submission_qc_checks.py

```diff
@@ -268,6 +284,154 @@ def check_if_browsable_files_entered_correctly_in_db(self, vcf_files):
             expected files: {vcf_files}
             missing files: {missing_files if missing_files else 'None'}"""
 
+    def clustering_check_report(self, target_assembly):
+        clustering_log_file = glob.glob(f"{self.path_to_data_dir}/53_clustering/logs/{target_assembly}_clustering.log")
```

Comment:
We should raise a ticket to get these logs back to `00_logs`

Reply:
Raised [EVA-3283](https://www.ebi.ac.uk/panda/jira/browse/EVA-3283)